senza/aws.py

```diff
@@ -24,6 +25,24 @@ def get_security_group(region: str, sg_name: str):
             raise
 
 
+def encrypt(region: str, KeyId: str, Plaintext: str, encode=False):
+    kms = boto3.client('kms', region)
+    encrypted = kms.encrypt(KeyId=KeyId, Plaintext=Plaintext)['CiphertextBlob']
+    if encode:
+        return base64.b64encode(encrypted).decode('utf-8')
+
+    return encrypted
+
+
+def list_kms_keys(region: str, describe=False):
+    kms = boto3.client('kms', region)
+    keys = list(kms.list_keys()['Keys'])
```

Comment:
Not very likely in practice, but you might want to use pagination here (this will list up to 100 keys and you can go as high as 1000 w/o pagination according to the docs).